sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/kafka/jaas/Jaas.java

```diff
@@ -0,0 +1,76 @@
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+package com.azure.spring.cloud.autoconfigure.implementation.kafka.jaas;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.azure.spring.cloud.autoconfigure.implementation.kafka.AzureKafkaConfigurationUtils.AZURE_CONFIGURED_JAAS_OPTIONS_KEY;
+import static com.azure.spring.cloud.autoconfigure.implementation.kafka.AzureKafkaConfigurationUtils.AZURE_CONFIGURED_JAAS_OPTIONS_VALUE;
+
+public class Jaas {
```

Comment:
what exactly this class is modeling?
I think it's trying to model something like this? 'com.ibm.auth.Module REQUIRED debug=true;'

what is this called in JaaS, I think it is not called 'Jaas' but something else, so maybe we should have a more accurate name with some java doc?